Index.html

```diff
@@ -1,179 +1,283 @@
+                        open_in_new
+                    </span>
+                </a>
+
+                <a class="buttons__inverse source__link see__pop">
+                    See Source
+                    <i class="fa fa-brands fa-github"></i>
+                </a>
+            </div>
+
+            <div class="navigation__buttons">
+                <button class="buttons__inverse nav__pop__prev nav__pop__down">
+                    <i class="fa fa-solid fa-arrow-left"></i>
+                    previous project
+                </button>
+
+                <button class="buttons__inverse nav__pop__next nav__pop__down">
+                    next project
+                    <i class="fa fa-light fa-arrow-right"></i>
+                </button>
+            </div>
+        </section>
+    </main>
+
+    <footer class="footer">
+        <button class="buttons footer__button">
+            <h4 class="footer__button__text">Get my resume</h4>
+            <img class="footer__button__image" src="./Contents/Resources/Icons/download_Resume.svg" alt="download my resume">
+        </button>
+
+        <ul class="socials">
+            <li href=""><img src="./Contents/Resources/Icons/ic_stackoverflow.svg" alt="go to my stackoverflow" class="social"></li>
+            <li href=""><img src="./Contents/Resources/Icons/ic_github.svg" alt="go to my github" class="social"></li>
+            <li href=""><img src="./Contents/Resources/Icons/ic_twitter.svg" alt="go to my twitter" class="social"></li>
+            <li href=""><img src="./Contents/Resources/Icons/ic_linkedin.svg" alt="go to my linkendin" class="social"></li>
+            <li href=""><img src="./Contents/Resources/Icons/ic_medium.svg" alt="go to my M" class="social"></li>
+            <li href=""><img src="./Contents/Resources/Icons/fingers.svg" alt="go to my M" class="social"></li>
         </ul>
-      </li>
-    </ul>
-  </section>
-  <form action="https://formspree.io/f/xeqdoajj" method="POST" id="contact-form">
-    <h1>Get Started</h1>
-    <p>I Prefer To Speak By My Gestures.</p>
-    <fieldset>
-      <label for="customer-name">
-        Your name <span class="errorText"></span>
-        <br />
-        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
-      </label>
-    </fieldset>
-    <fieldset>
-      <label for="customer-email">
-        Your email <span class="errorText"></span>
-        <br />
-        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
-      </label>
-    </fieldset>
-    <span class="errorText"></span>
-    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
-      required></textarea>
-    <span class="errorText" id="submitError"></span>
-    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
-  </form>
```

Comment:
Kindly, I suggest you follow the template as it is, to make your project look professional. It's highly recommended to show the label for each input and text-area.
Kindly make sure you fix these changes to make your project match the template.

>### The Figma Design
>
><img width="166" alt="Screenshot 2023-04-28 at 15 37 15" src="https://user-images.githubusercontent.com/91237525/235238390-1f246616-be67-4815-94d7-620832b66109.png">
>
>
>### Your Design
>
><img width="186" alt="Screenshot 2023-04-28 at 15 37 06" src="https://user-images.githubusercontent.com/91237525/235238467-03fc6e28-69c0-4a7b-9035-8de8fa3b36b5.png">